### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by athulp01**

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
Please drop the tkinter commit. Also add the `pyenv` folder to `.gitignore`

File: README.md [hunk 1]
```diff
@@ -10,3 +10,13 @@ A simple game built using Python Pygame Library.
 </dl>
 
 ![Screenshot](https://cloud.githubusercontent.com/assets/18749480/16371338/ad6861c4-3c63-11e6-85eb-fa8af1c726d4.png "Screenshot")
+
+# Installation instructions:
+
+1. Download the repo or clone it to your computer and change to that directory.
+2. Create a virtualenv using  `mkdir path/snake && virtualenv path/snake`
```
Comment (jajodiaraghav) [inline README.md]: Change this line to:
Create a virtualenv using `virtualenv pyenv`

File: README.md [hunk 2]
```diff
@@ -10,3 +10,13 @@ A simple game built using Python Pygame Library.
 </dl>
 
 ![Screenshot](https://cloud.githubusercontent.com/assets/18749480/16371338/ad6861c4-3c63-11e6-85eb-fa8af1c726d4.png "Screenshot")
+
+# Installation instructions:
+
+1. Download the repo or clone it to your computer and change to that directory.
```
Comment (jajodiaraghav) [inline README.md]: Replace this with:
Clone the repository and `cd` in to the folder.

**Comment by athulp01** [comment]
how to drop the tkinter commit? @jajodiaraghav

**Comment by jajodiaraghav** [comment]
Hi @athulpn You do not have to create new PRs every time you have to make changes to commits. All you have to do is this: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

From further PRs, try to use `git commit --amend` to amend changes to last commit you made.

**Comment by jajodiaraghav** [comment]
Feel free to ask in case you are stuck. All you have to do is do `git rebase -i head~4` since you have 4 commits in this PR
Please go through the link for instructions.

**Comment by athulp01** [comment]
I messed things up. I used web interface to update the markdown

**Pull request merged**